tests/provider.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    indykite = {
+      source = "indykite/indykite"
+      version = "0.2.1"
```

Comment:
Is version relevant since the plugin is build locaaly ?

Reply:
I think it is. Why build locally?
May be we could use latest

Reply:
we can not use latest, the pipelien runs before the new plugin changes has been publiched the Terraform registry.
So to test if the changes made to the plugin are valid it has to be build locally

Reply:
or use something like
`terraform {
  required_providers {
    example = {
      version = "~> 1.0.0"
      source  = "path_to_local_binary"
    }
  }
}`